sale_validity_auto_cancel/models/res_partner.py

```diff
@@ -0,0 +1,11 @@
+# Copyright 2023 ForgeFlow S.L.
+# Copyright 2024 OERP Canada <https://www.oerp.ca>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    is_keep_so = fields.Boolean(string="Keep Expired SO")
```

Comment:
I would recommend a more descriptive technical name for the field taking into account the module naming. For example it could be like 'auto_cancel_expired_so', which would be True for all partners by default.

Reply:
Hi @JordiMForgeFlow 

Thank you for your reply and suggestion, I'll update accordingly and will request for review.

Thank you